NFT-Collection.md

```diff
@@ -751,7 +751,7 @@ export default function Home() {
     if (!presaleStarted) {
       return (
         <div>
-          <div className={styles.description}>Presale hasnt started!</div>
+          <div className={styles.description}>Presale hasn't started!</div>
```

Comment:
Since this is inside a rendering tag, you should do `hasn&#39;t`. Please test this out first.